packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -130,208 +153,238 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
           <SiteNavAnchor to="/">Home</SiteNavAnchor>
         </SiteNavItem>
         <SiteNavItem>
-          <SiteNavButton
+          <SideNavDisclosureButton
             onClick={() => setgettingStartedOpen(!gettingStartedOpen)}
             isOpen={gettingStartedOpen}
             aria-expanded={gettingStartedOpen}
           >
             Getting Started
-            <SiteNavAnchorArrow isOpen={gettingStartedOpen} />
-          </SiteNavButton>
-          <SiteNavNestList isOpen={gettingStartedOpen}>
-            <SiteNavItem>
-              <SiteNavAnchor to="/getting-started/engineering">Engineering Guidelines</SiteNavAnchor>
-            </SiteNavItem>
-            <SiteNavItem>
-              <SiteNavAnchor to="/getting-started/design">Design Guidelines</SiteNavAnchor>
-            </SiteNavItem>
-            <SiteNavItem>
-              <SiteNavAnchor to="/getting-started/working-guide">How to work with us</SiteNavAnchor>
-            </SiteNavItem>
-            <SiteNavItem>
-              <SiteNavAnchor to="/getting-started/about-paste">About Paste</SiteNavAnchor>
-            </SiteNavItem>
-            <SiteNavItem>
-              <SiteNavAnchor to="/getting-started/faqs">FAQs</SiteNavAnchor>
-            </SiteNavItem>
-          </SiteNavNestList>
+          </SideNavDisclosureButton>
+          <DisclosurePrimitiveContent visible={gettingStartedOpen}>
```

Comment:
I see, so yeah, we should use all the bits from the disclosure primitive. What is happening is you're losing all the accessibility goodness like `aria-controls` on the button, which points to the id of this div when using all the pieces together

Reply:
Ah. I see. I refactored this using the `DisclosurePrimitive`.